cmd/cedana-helper/install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Check whether git is already installed
+if ! command -v git &> /dev/null; then
+    yum install -y git
+fi
+
+# Download and install Go
+wget https://go.dev/dl/go1.22.0.linux-arm64.tar.gz
+rm -rf /usr/local/go
+tar -C /usr/local -xzf go1.22.0.linux-arm64.tar.gz
+
+# Add Go binary directory to PATH
+export PATH=$PATH:/usr/local/go/bin
+
+# Add export statement to ~/.bashrc
+echo "export PATH=$PATH:/usr/local/go/bin" >> /root/.bashrc
+
+# Install required packages
+yum install -y libnet-devel protobuf protobuf-c protobuf-c-devel protobuf-compiler protobuf-devel protobuf-python libnl3-devel libcap-devel
+yum group install -y "Development Tools"
+
+# Clone CRIU repository and build
+git clone https://github.com/checkpoint-restore/criu.git
```

Comment:
what's the frequency of this install.sh operation? can you time it? concerned building from source is going to take too long